php/http_request.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+require 'vendor/autoload.php';
```

Comment:
you have to have `composer.json` file in the project dir to be able require `vendor/autoload.php` but I don't see it. But you can include preinstalled libs from the `require '/libs/vendor/autoload.php'`

Reply:
added composer.json to project